docs/spark-procedures.md

```diff
@@ -721,14 +721,15 @@ Creates a view that contains the changes from a given table.
-|`identifier_columns`| | array<string> | The list of identifier columns to compute updates. If the argument `compute_updates` is set to true and `identifier_columns` are not provided, the table’s current identifier fields will be used to compute updates. |
-|`remove_carryovers`| | boolean | Whether to remove carry-over rows (see below for more information). Defaults to true.                                                                                                                                 |
+| Argument Name        | Required? | Type                | Description                                                                                                                                                                                                          |
+|----------------------|----------|---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `table`              | ✔️ | string              | Name of the source table for the changelog                                                                                                                                                                           |
+| `changelog_view`     |   | string              | Name of the view to create                                                                                                                                                                                           |
+| `options`            |   | map<string, string> | A map of Spark read options to use                                                                                                                                                                                   |
+| `net_changes`        | | boolean             | Whether to output net changes (see below for more information). Defaults to false. |
+| `compute_updates`    | | boolean             | Whether to compute pre/post update images (see below for more information). Defaults to false.                                                                                                                       | 
+| `identifier_columns` | | array<string>       | The list of identifier columns to compute updates. If the argument `compute_updates` is set to true and `identifier_columns` are not provided, the table’s current identifier fields will be used to compute updates. |
+| `remove_carryovers`  | | boolean             | Deprecated. Whether to remove carry-over rows (see below for more information). Defaults to true.                                                                                                                    |
```

Comment:
Up to you, but should we add version for deprecation, and recommendation what to use instead?

Reply:
fixed